Magento_Cms/web/css/source/pages/_404.less

```diff
@@ -0,0 +1,104 @@
+@import 'https://fonts.googleapis.com/css?family=Dosis:300,400,700,800';
+
+//
+//  Common
+//  _____________________________________________
+
+& when (@media-common = true) {
+    .cms-no-route {
+        background: #1d2d44ff;
+        color: @color-general-white;
+        font-family: Dosis, sans-serif;
+
+        h1 {
+            color: @color-general-white;
+            margin: 35px 0 15px;
+            font-size: 55px;
```

Comment:
don't forget to use .lib-font-size for font-size and .lib-line-height for line-height (only in PX) to convert PX to REM. it's important for a11y